src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/BreakpointComplications.java

```diff
@@ -319,7 +343,8 @@ static String getInsertedSequence(final AlignedAssembly.AlignmentInterval curren
         }
     }
 
-    // TODO: 03/03/17 this complicated tandem duplication case is not exactly reproducible (e.g. '+' and '-' strand may give slightly different results by this treatment)
+    // TODO: 03/03/17 this complicated tandem duplication case is not exactly reproducible
```

Comment:
Still a valid concern. Updated with more clarification. And moved comment for its ctor to this class.